File: data/src/main/java/home/oleg/placesnearme/data/model/DetailedVenueHistory.java
package home.oleg.placesnearme.data.model;


import android.arch.persistence.room.Embedded;

public class DetailedVenueHistory {

    private Long createdAt;

    @Embedded
    private DetailedVenueWithPhotos detailedVenueWithPhotos;

    public Long getCreatedAt() {
        return createdAt;
    }

    public void setCreatedAt(Long createdAt) {
        this.createdAt = createdAt;
    }

    public DetailedVenueWithPhotos getDetailedVenueWithPhotos() {
        return detailedVenueWithPhotos;
    }

    public void setDetailedVenueWithPhotos(DetailedVenueWithPhotos detailedVenueWithPhotos) {
        this.detailedVenueWithPhotos = detailedVenueWithPhotos;
    }
}

File: core-data/src/main/java/home/oleg/placesnearme/data/model/DetailedVenueHistory.kt
package home.oleg.placesnearme.data.model

import androidx.room.Embedded
import androidx.room.Entity

@Entity
data class DetailedVenueHistory(
        val createdAt: Long,
        @Embedded
        val detailedVenueWithPhotos: DetailedVenueWithPhotos?)
